scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -315,6 +315,99 @@ def test_rotvec_calc_pipeline():
     assert_allclose(Rotation.from_rotvec(rotvec).as_rotvec(), rotvec)
 
 
+def test_from_1d_single_mrp():
+    mrp = [0, 0, 0.41421356]
+    expected_quat = np.array([0, 0, 0.70710678, 0.70710678])
+    result = Rotation.from_mrp(mrp)
+    assert_array_almost_equal(result.as_quat(), expected_quat)
+
+
+def test_from_2d_single_mrp():
+    mrp = [[0, 0, 0.41421356]]
+    expected_quat = np.array([[0, 0, 0.70710678, 0.70710678]])
+    result = Rotation.from_mrp(mrp)
+    assert_array_almost_equal(result.as_quat(), expected_quat)
+
+
+def test_from_generic_mrp():
+    mrp = [
+            [1, 2, 2],
+            [1, -1, 0.5],
+            [0, 0, 0]
+            ]
+    expected_quat = np.array([
+        [0.2, 0.4, 0.4, -0.8],
+        [0.61538462, -0.61538462, 0.30769231, -0.38461538],
+        [0, 0, 0, 1]
+        ])
+    assert_array_almost_equal(
+            Rotation.from_mrp(mrp).as_quat(),
```

Comment:
Questionable formatting here as well.

Reply:
Resolved, along with others. Do you guy prefer to hit the "resolve conversation" button when a comment is addressed or just leave it?